packages/commonwealth/client/scripts/views/components/component_kit/CWRelatedCommunityCard/CWRelatedCommunityCard.scss

```diff
@@ -0,0 +1,105 @@
+@import '../../../../../styles/shared.scss';
+
+.RelatedCommunityCard {
+  padding: 24px;
+  display: flex;
+  align-items: center;
+  width: 506px;
```

Comment:
probably this should not have fixed size, eg because of lack of responsiveness on smaller screens 
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/379be920-bd42-497b-9712-ad22e37344c9)